qiskit_sphinx_theme/pytorch_base/static/css/theme.css

```diff
@@ -11879,19 +11878,19 @@ article.pytorch-article .tutorials-callout-container .btn.callout-button a {
     margin-left: 30px;
   }
 }
-.pytorch-content-left .main-content {
-  padding-top: 3rem;
+.pytorch-content-left .main-content ul.simple {
+  padding-bottom: 20px;
 }
```

Comment:
Why this change?

Reply:
i messed up the merge conflict fix my bad

Reply:
i just undid that change